app/core/migrations/0004_auto_20210910_1106.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.1.15 on 2021-09-10 11:06
```

Comment:
In a real world case you probably want to squash all those migration files before merging, to not have the history of the multiple iterations of the models within the same PR.

Reply:
Gotcha